library/src/main/java/com/readystatesoftware/chuck/internal/ui/error/ErrorListFragment.java

```diff
@@ -97,6 +98,24 @@ public boolean onOptionsItemSelected(MenuItem item) {
         }
     }
 
+    private void askForConfirmation() {
+        new AlertDialog.Builder(getContext())
+            .setTitle(R.string.chuck_clear)
+            .setMessage(R.string.chuck_clear_error_confirmation)
+            .setPositiveButton(R.string.chuck_clear, new DialogInterface.OnClickListener() {
+                @Override
+                public void onClick(DialogInterface dialog, int which) {
+                    getContext().getContentResolver().delete(ChuckContentProvider.ERROR_URI, null, null);
+                }
+            })
+            .setNegativeButton(R.string.chuck_cancel, new DialogInterface.OnClickListener() {
```

Comment:
You can replace here with

```suggestion
            .setNegativeButton(R.string.chuck_cancel, null)
```

And remove all the lines below

Reply:
Ok